### ISSUE TITLE
New license request: carnegie-mellon

### ISSUE THREAD

**Issue by yarda** [spdx/license-list-XML#2321]
### How license meets inclusion principles

https://gitlab.com/fedora/legal/fedora-license-data/-/issues/445

> This is pretty close to HPND but has some minor differences.

### License Name

No idea

### Suggested short identifier

No idea

### License or Exception?

license

### URL to license text

https://github.com/ppp-project/ppp/blob/master/pppd/plugins/radius/COPYRIGHT#L6-L17

### OSI Status

I don't know

### License author or steward

Unknown

### URL to project(s) that use license

https://github.com/paulusmack/ppp

### paste text of license here

```
Permission to use, copy, modify, and distribute this software for any
purpose and without fee is hereby granted, provided that this
copyright and permission notice appear on all copies and supporting
documentation, the name of Roaring Penguin Software Inc. not be used
in advertising or publicity pertaining to distribution of the program
without specific prior permission, and notice be given in supporting
documentation that copying and distribution is by permission of
Roaring Penguin Software Inc..

Roaring Penguin Software Inc. makes no representations about the
suitability of this software for any purpose.  It is provided "as is"
without express or implied warranty.
```

**Comment by jlovejoy** [comment]
close to `HPND` except:

1. omits "and its documentation in the first part, which is similar to #2238
2. requirement to "provided that this copyright and this permission notice appear in all copies and supporting documentation" which is similar to #2319 but with slightly different wording
3. adds an obligation that "and notice be given in supporting documentation that copying and distribution is by permission of Roaring Penguin Software Inc.." which is not in any of the other HPND variants we've seen

Given the third item in particular, I think this is probably a new license

**Comment by jlovejoy** [comment]
and note, the link provided above shows 3 "variants" of this license - the same substantively, but each with a different name (i.e., "Roaring Penguin") - so can't use that in the name.

Maybe `HPND-permission-notice` ??

**Comment by nmav** [comment]
> @jlovejoy This text appears to be the same as #2286 (which was submitted for the next two entries below it).
> 
> GIven that, maybe this one and the ones in #2286 should all be merged as something like `HPND-PPP` (noting the reference to the project in which these all seem to appear)?

Using the "field" or project name that these licenses appear makes sense, especially if that is a project that the source originates  from. That's not the case with the PPP project and its radius plugin, thus using PPP in the name may confuse as PPP itself is not really under this license -only one plugin that can be removed/disabled-.

The earlier reference to this license I could find is a project called [radiusclient-ng](http://developer.berlios.de/projects/radiusclient-ng/) which itself seems to be derived from some Livingston radius product that has disappeared from the internet. As such I think that the proposal in https://github.com/spdx/license-list-XML/issues/2286#issuecomment-1919236529 will be more accurate, and less confusing.

**Comment by swinslow** [comment]
Thank you @nmav! Makes sense and appreciate the background on this one.

@jlovejoy Given this, perhaps we do go with `HPND-Fenneberg-Livingston` for #2286, and then once it's merged, close this one as a match for that?

**Comment by jlovejoy** [comment]
given we have resolved this as same license as #2286, I'm going to close this issue
